container-images/webpack.config.js

```diff
@@ -0,0 +1,65 @@
+const webpack = require('webpack')
```

Comment:
If this is identical to `aws-lambda/webpack.config.js` we could probably just have one copy in the root folder and reference it from there via `require('../../webpack.config')`

Reply:
They're not quite the same - the container-images webpack config does not use the zip plugin.

Reply:
Fair. Would be pretty straight forward to make zip an option, but not much benefit ATM.